unoplat-code-confluence-frontend/AGENTS.md

```
@@ -115,4 +115,7 @@ See [`business_domain_references.md`](./business_domain_references.md) for the s
 ## App Interfaces
 See [`app_interfaces.md`](./app_interfaces.md) for the canonical interface and endpoint reference.
 
+## Architecture
```

markdownlint-cli2 (0.23.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)